python/core/auto_generated/qgsvectorlayer.sip.in

```diff
@@ -1042,13 +1042,15 @@ if the geometry type of the new data source matches the current geometry type of
     virtual QString loadDefaultStyle( bool &resultFlag /Out/ ) ${SIP_FINAL};
 
 
-    QgsVectorLayerFeatureCounter *countSymbolFeatures();
+    QgsVectorLayerFeatureCounter *countSymbolFeatures( bool withFids = false );
```

Comment:
I'd introduce the term "symbol" somehow in this flag.
```suggestion
    QgsVectorLayerFeatureCounter *countSymbolFeatures( bool storeSymbolFids = false );
```

Reply:
should be done.